docs/pages/nft/[nftAddress].tsx

```diff
@@ -106,15 +87,31 @@ export default function NftPage({
           <div>
             <h1>{nft.name}</h1>
 
-            <ValueList attributes={attributes} />
+            <div>
+              <h2 className="text-secondary">On-Chain Metadata</h2>
+              <ValueList
```

Comment:
`ValueList` should probably take in a list rather than an object so that we can decide the order